packages/flutter/lib/src/gestures/eager.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'arena.dart';
+import 'events.dart';
+import 'recognizer.dart';
+
+/// A gesture recognizers that eagerly claims victory in all gesture arenas.
+///
+/// This is typically passed in [AndroidView.gestureRecognizers] in order to immediately dispatch
+/// all touch events inside the view bounds to the embedded Android view.
+class EagerGestureRecognizer extends OneSequenceGestureRecognizer {
```

Comment:
why do you want this to be a OneSequenceGestureRecognizer? it seems like that drags in a bunch of logic you don't need, given how trivial this recognizer is.

Reply:
Right now AndroidView takes only OneSequenceGestureRecognizers as these are the only ones who currently support gesture teams.
I have a TODO for this (see #20953)